Dockerfile

```
@@ -0,0 +1,14 @@
+FROM python:3.11-slim
+
+WORKDIR /app
+
+RUN apt-get update && apt-get install -y git
+
+COPY tools/ tools/
+COPY requirements.txt .
+COPY process_report/process_report.py process_report/
+COPY process_report/institute_map.json process_report/
+
+RUN pip install -r requirements.txt
```

Always handle dependency installation first; this allows your container runtime to cache this step so that you don't need to re-install requirements every time you modify your code and rebuild the image.

```suggestion
COPY requirements.txt .
RUN pip install -r requirements.txt

COPY tools/ tools/
COPY process_report/process_report.py process_report/
COPY process_report/institute_map.json process_report/
```